resources/snippets.json

```
@@ -401,6 +401,12 @@
     "code": ".main-trackList-disabled{background:#f004}.main-trackList-disabled:focus-within,.main-trackList-disabled:hover{background:#f005}.main-trackList-disabled.main-trackList-selected,.main-trackList-disabled.main-trackList-selected:hover{background:#f006}",
     "preview": "resources/assets/snippets/More-Visible-Unplayable-Tracks"
   },
+  {
+    "title": "[Global Nav Bar] remove scrolling overflow",
+    "description": "Removes the max visible width restrictions of the 'sidebar' items. Useful if you have more than the scrolling limit but not as much that it will overflow with the searchbar",
```

Some wording suggestions because it sounds really confusing at first glance.
```suggestion
    "description": "Removes the max visible width restrictions of the 'sidebar' items. Useful if you have more than the scrolling limit but not so much that it will interfere with the search bar",
```